scripts/rest_examples/reservation-create-aws.http

```diff
@@ -4,7 +4,7 @@ Content-Type: application/json
 X-Rh-Identity: {{identity}}
 
 {
-  "name": "amz-linux-us-east-1",
+  "names": ["amz-linux-us-east-1"],
   "source_id": "1",
   "image_id": "ami-05fa00d4c63e32376",
   "amount": 1,
```

Comment:
If you pass an array with exactly one name, it works as previously - the launchInstance function will actually proceed because it can tag all of them. We should probably update the check condition, it should only do it if they are ALL the same, not when there is only one tho.